mitosheet/src/mito/components/footer/GraphTabContextMenu.tsx

```diff
@@ -58,13 +71,22 @@ export default function GraphSheetTabContextMenu(props: {
     }
 
     const openExportGraphTaskpaneTab = async (): Promise<void> => {
+        const existingParams = await getParamsForExistingGraph(props.mitoAPI, props.graphID);
+
+        if (existingParams === undefined) {
+            return;
+        }
         props.setUIState(prevUIState => {
             return {
                 ...prevUIState,
                 currOpenTaskpane: {
                     type: TaskpaneType.GRAPH, 
-                    graphID: props.graphID, 
-                    graphSidebarTab: GraphSidebarTab.Export
+                    graphSidebarTab: GraphSidebarTab.Export,
+                    openGraph: {
+                        type: 'existing_graph',
```

Comment:
Are these types defined anywhere? I feel like they're long enough that it's worth making sure they're correct through typechecking.